common/servlet/src/main/java/io/apiman/common/servlet/ApimanCorsFilter.java

```diff
@@ -57,16 +57,20 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         HttpServletRequest httpReq = (HttpServletRequest) request;
         HttpServletResponse httpResp = (HttpServletResponse) response;
 
-        if (isPreflightRequest(httpReq) && originIsAllowed(httpReq)) {
-            httpResp.setHeader("Access-Control-Allow-Origin", httpReq.getHeader("Origin")); //$NON-NLS-1$ //$NON-NLS-2$
+        // Get system property here
+        String[] allowOriginsA = System.getProperty("apiman-manager-ui.allowed-cors-origins", "*").split(",");
```

Comment:
Please compare with existing code: https://github.com/apiman/apiman/blob/5c85f0e1a19e13804f7dc2a1634ac0525b6a57ff/common/servlet/src/main/java/io/apiman/common/servlet/ApimanCorsFilter.java#L37-L42

Reply:
Nice approach.
I will do something like that.
> It would be nice to read the value out of the apiman.properties instead.
I'm not a pro in JAVA, more and less code monkey :P
Where can i find an exemple of reading properties of `apiman.properties` ?
I haven't find one.

Reply:
I think a good start would be here: https://github.com/apiman/apiman/blob/master/manager/ui/war/src/main/java/io/apiman/manager/ui/server/UIConfig.java

Reply:
I have added all, your approach and the configuration.